hdf5.spec

```
@@ -15,13 +15,12 @@
 %global hdf5_major 1
 %global hdf5_minor 13
 %global hdf5_bugfix 0
-%global hdf5_prerelease rc5
-%global hdf5_tag %{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}%{?hdf5_prerelease:-%{hdf5_prerelease}}
+%global hdf5_tag %{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: %{hdf5_major}.%{hdf5_minor}.%{hdf5_bugfix}%{?hdf5_prerelease:~%{hdf5_prerelease}}
-Release: 5%{?commit:.git%{shortcommit}}%{?dist}
+Version: %{hdf5_major}.%{hdf5_minor}.%{hdf5_bugfix}
```

Since `hdf5_prerelease` is undefined, this line doesn't need to change.

```suggestion
Version: %{hdf5_major}.%{hdf5_minor}.%{hdf5_bugfix}%{?hdf5_prerelease:~%{hdf5_prerelease}}
```